serving/src/main/java/feast/auth/credentials/CoreAuthenticationProperties.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright 2018-2020 The Feast Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package feast.auth.credentials;
+
+import feast.common.validators.OneOfStrings;
+import java.util.Map;
+
+public class CoreAuthenticationProperties {
```

Comment:
Might be out of scope but is it possible to get some/all of these properties from Core instead of forcing the user to specify them manually. Serving already has a dependency on Core running when starting.

Reply:
Wouldn't it be nice if there wasn't a connection dependency though?  Maybe these properties are only useful for core but if there is a direct need for Serving to know about them, I would argue that the dependency to them should be direct.

I guess the argument for getting them from core would be if serving is just delegating and it is feasible that serving could connect to multiple core instances someday and the settings would be specific to each.

Reply:
I want to see if we can remove the dependency on Core, not make matters worse. Further coupling these systems is not a good idea in my opinion.

Reply:
we can do that but not in this PR.  These are just options to authenticate against core which is currently broken because either authentication is not turned on when you think it is or when it is turned on serving doesn't start. 

Maybe we need a new issue for the dependency on core.  I'm curious what you think that would like like.  I think it's weird that the client has two servers to connect to.  I would rather see one API that serves all clients and put job mgmt in it's own microservice.  In that way the core API that serving is currently using becomes part of serving.